lib/pliny/log.rb

```diff
@@ -4,6 +4,10 @@ def log(data, &block)
       log_to_stream(stdout || $stdout, merge_log_contexts(data), &block)
     end
 
+    def log_without_context(data, &block)
```

Comment:
I can see arguments for both approaches. I get the motivation for keeping things explicit here. However, I do wonder if there's ever a log line that you'd want to emit without the `default_context`? Also, passing the entire `config` object into something that holds log lines seems like it could potentially leak creds.